astroplan/core.py

```diff
@@ -161,6 +163,46 @@ def altaz(self, time, target=None, obswl=None):
                 coordinate = target
             return coordinate.transform_to(altaz_frame)
 
+    def parallactic_angle(self, time, target):
```

Comment:
For e.g. `separation` the functions exsit because sometimes you want to do that stuff on a low-level class *or* a `SkyCoord`.  That is, the same function can be used on multiple types of objects.  Here I think the only thing you'd use it on is `Observer`, right?  So I'd say stick with the principal that "there should be only one obvious way to do it" and just leave it as a method.